opentelemetry-jaeger/Cargo.toml

```diff
@@ -43,7 +43,7 @@ tokio = { workspace = true, features = ["net", "sync"], optional = true }
 wasm-bindgen = { version = "0.2", optional = true }
 wasm-bindgen-futures = { version = "0.4.18", optional = true }
 
-tonic = { workspace = true, optional = true }
+tonic = { workspace = true, optional = true, default-features = true }
```

Comment:
Instead of doing this, it would be nice to enable the minimal set of features actually needed to make this compile.